components/browser/menu/src/main/java/mozilla/components/browser/menu/BrowserMenu.kt

```diff
@@ -27,34 +26,33 @@ class BrowserMenu internal constructor(
     private var currentPopup: PopupWindow? = null
     private var menuList: RecyclerView? = null
 
+    @Suppress("UNUSED_PARAMETER")
     @SuppressLint("InflateParams")
     fun show(anchor: View, orientation: Orientation = Orientation.DOWN): PopupWindow {
```

Comment:
If you agree with this approach, I don't think we will need to provide an `orientation` parameter anymore. As the popup will be responsible for laying out the orientation and dimension. We can delete everything related to it.